src/main/java/exceptions/NotFoundInArrayList.java

```diff
@@ -0,0 +1,7 @@
+package exceptions;
+
+public class NotFoundInArrayList extends  Exception{
```

Comment:
Also please read about Checked vs Unchecked exceptions and avoid creating custom exceptions derived from Exception class